r2/r2/controllers/ipn.py

```diff
@@ -590,9 +590,9 @@ def finish(self, parameters, txn_id,
                         "supports the site and makes future development "
                         "possible. For example, one month of reddit gold "
                         "pays for 5 instance hours of reddit's servers.")
-            message += "\n\n" + strings.gold_benefits_msg
+            message += "\n\n" + _(strings.gold_benefits_msg)
```

Comment:
Oh I just saw that you untranslated it down below. Hmm might need to see if there's a way to untranslate it or just create a new variable.